app/js/disease-associations-table-directive.js

```diff
@@ -398,7 +400,7 @@ angular.module('cttvDirectives')
                                     "target.gene_info.name"],
                                 from: from
                             };
-                            if (scope.target) {
+                            if (scope.target.length) {
```

Comment:
No, you are right, this should be
if (scope.target && scope.target.length)
changing it now
thanks!